dicompylercore/dvhcalc.py

```diff
@@ -228,8 +232,15 @@ def _calculate_dvh(structure,
                 logger.warning('Dose plane not found for %s.' +
                                ' Using %s to calculate contour volume.',
                                z, origin_z)
+                # Use dummy dose grid
```

Comment:
Can you update this comment on why we want to use the dummy dose grid, just for reference?

Reply:
Sure